test/lib/usd/translators/UsdImportInstancesTest/ComplexScenario_optimized.usda

```diff
@@ -0,0 +1,146 @@
+#usda 1.0
+(
+    defaultPrim = "pCube1"
+    metersPerUnit = 0.01
+    upAxis = "Y"
+)
+
+def Xform "pCube1" (
```

Comment:
One small nit-pick with the type names here, but it looks like we ended up with type names everywhere. I would think the majority of the time when you have a prim referencing other prims, you likely want a typeless def for the prim doing the referencing. I *think* in most cases below, the type names between the referencer and the referenced agree, so I don't think it's a practical issue here. Typeless defs let the type of the referenced prim shine through to the referencer whatever that type may be, so you don't have to match the types yourself. Specifying a different type on the referencer will actually change the type of the composed prim, which is pretty rarely what you intend.